substrate/frame/paged-list/src/paged_list.rs

```diff
@@ -60,10 +62,10 @@ pub type ValueIndex = u32;
 /// as long as there are elements in the page and there are pages in storage. All elements of a page
 /// are loaded once a page is read from storage. Iteration then happens on the cached elements. This
 /// reduces the number of storage `read` calls on the overlay. **Appending** to the list happens by
-/// appending to the last page by utilizing [`sp_io::storage::append`]. It allows to directly extend
+/// appending to the last page by utilizing [`storage::append`]. It allows to directly extend
```

Comment:
This breaks the intra code link.

```suggestion
/// appending to the last page by utilizing [`storage::append`](frame::deps::sp_io::storage::append). It allows to directly extend
```